package.json

```diff
@@ -47,6 +47,7 @@
   },
   "devDependencies": {
     "@google-cloud/nodejs-repo-tools": "^3.0.0",
+    "ava": "0.25.0",
```

Comment:
Greetings!  We're using `mocha` now for all of our tests.  Also - I am guessing you wanted this up in `samples/package.json` :)

Reply:
How does mocha work? I dropped ava in both package.json, and got below error when run command "npm run lint ./samples/system-test/*.js -- --fix":

~/github/nodejs-asset/samples/system-test/quickstart.test.js
  19:22  error  "ava" is extraneous  node/no-extraneous-require

✖ 1 problem (1 error, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/asset@0.1.0 lint: `eslint src/ samples/ system-test/ test/ "./samples/system-test/quickstart.test.js" "--fix"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/asset@0.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!    ~/.npm/_logs/2018-11-15T05_47_31_397Z-debug.log